src/AdoNet/Orleans.AdoNet.Core/Orleans.AdoNet.Core.csproj

```diff
@@ -0,0 +1,32 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <PackageId>Microsoft.Orleans.AdoNet.Core</PackageId>
```

Comment:
Is there any changes to the source from this project compared to the files stored in the shared folder (appart from the suppression of the compiler directive and c# modernization)?

Not sure if it's github or if we are losing the change history with this move.

Reply:
> Is there any changes to the source from this project compared to the files stored in the shared folder (appart from the suppression of the compiler directive and c# modernization)?

No changes are intended apart from:

1) Removing compiler directives.
2) Moving shared code to a core project.

This includes no great intention for C# modernization just yet. The objective is to lessen the review burden as there are already 89 files to look at. There were a few small bits that were refactored now but only to apply analyser fixes, such as the `switch` expression change in `AdoNetFormatProvider`.

The lack of further refactoring is why the `InternalsVisibleTo` attributes are there for now. The original provider code was tightly coupled to the shared code via linked files. This coupling did not change as of this PR, it has only been made formal.

The next PR will deal with moving the core classes into DI so the forward dependencies can be removed.

> Not sure if it's github or if we are losing the change history with this move.

I did use `git mv` to keep file history where appropriate and I did commit each move on its own. This can be verified in the individual "moved file" commits. I'm not sure why Github is not showing more files as moved.